backend/ops_api/tests/ops/features/validate_draft_budget_lines.feature

```diff
@@ -0,0 +1,138 @@
+
+    When I have a BLI in DRAFT status
+    And I submit a BLI to move to IN_REVIEW status
+
+    Then I should get an error message that the BLI's Agreement must have a valid Agreement Reason
+
+  Scenario: Valid Agreement Reason - NEW_REQ does not have an Incumbent
+    Given I am logged in as an OPS user
+    And I have an Agreement with an AgreementReason = NEW_REQ and an Incumbent
+
+    When I have a BLI in DRAFT status
+    And I submit a BLI to move to IN_REVIEW status
+
+    Then I should get an error message that the BLI's Agreement cannot have an Incumbent if it has an Agreement Reason of NEW_REQ
+
+  Scenario: Valid Agreement Reason - RECOMPETE and LOGICAL_FOLLOW_ON requires an Incumbent
+    Given I am logged in as an OPS user
+    And I have an Agreement with an AgreementReason = RECOMPETE or LOGICAL_FOLLOW_ON and has a NULL or empty string Incumbent
+
+    When I have a BLI in DRAFT status
+    And I submit a BLI to move to IN_REVIEW status
+
+    Then I should get an error message that the BLI's Agreement must have an Incumbent if it has an Agreement Reason of RECOMPETE or LOGICAL_FOLLOW_ON
+
+  Scenario: Valid Project Officer
+    Given I am logged in as an OPS user
+    And I have an Agreement without a Project Officer
+
+    When I have a BLI in DRAFT status
+    And I submit a BLI to move to IN_REVIEW status
+
+    Then I should get an error message that the BLI's Agreement must have a Project Officer
+
+  Scenario: Valid Team Members
+    Given I am logged in as an OPS user
+    And I have an Agreement without any Team Members
+
+    When I have a BLI in DRAFT status
+    And I submit a BLI to move to IN_REVIEW status
+
+    Then I should get an error message that the BLI's Agreement must have at least one Team Member
+
+  Scenario: Valid Description
+    Given I am logged in as an OPS user
+    And I have a valid Agreement
+
+    When I have a BLI in DRAFT status without a Description
+    And I submit a BLI to move to IN_REVIEW status
+
+    Then I should get an error message that the BLI must have a Description
+
+
+  Scenario: Valid Need By Date
+    Given I am logged in as an OPS user
+    And I have a valid Agreement
+
+    When I have a BLI in DRAFT status without a Need By Date
+    And I submit a BLI to move to IN_REVIEW status
+
+    Then I should get an error message that the BLI must have a Need By Date
+
+  Scenario: Valid CAN
+    Given I am logged in as an OPS user
+    And I have a valid Agreement
+
+    When I have a BLI in DRAFT status without a CAN
+    And I submit a BLI to move to IN_REVIEW status
+
+    Then I should get an error message that the BLI must have a CAN
+
+  Scenario: Valid Amount
+    Given I am logged in as an OPS user
+    And I have a valid Agreement
+
+    When I have a BLI in DRAFT status without an Amount
+    And I submit a BLI to move to IN_REVIEW status
+
+    Then I should get an error message that the BLI must have an Amount
+
+  Scenario: Valid Agreement
```

Comment:
Not sure it matters on the ordering for the `.feature` it self, but for logical reading, I'd think the `Scenario: Valid Agreement` should be near the top? Only because as I read each `Scenario` prior to this, I asked myself "what is considered a "valid agreement" in all the `Given` clauses.